client/src/components/requester_task/task.html

```diff
@@ -1,8 +1,8 @@
 <div id="requester-task" ng-controller="RequesterTaskCtrl as ctrl">
   <nav-bar></nav-bar>
   
-  <div ui-view="location"></div>
-  <div ui-view="item"></div>
-  <div ui-view="time_price"></div>
-  <div ui-view="confirm"></div>
+  <div ng-show = "ctrl.location" ui-view="location"></div>
+  <div ng-show = "ctrl.item" ui-view="item"></div>
+  <div ng-show = "ctrl.timePrice" ui-view="time_price"></div>

```

Comment:
The time_price was written by Darrin, when he was setting up client side routing and the view is named time_price in app.js as well, so I think it would be best to leave it as is. I  don't want to change timePrice to be underscore though, because then my controller method naming will become inconsistent, so I think we should just leave both as they are.